ui/src/components/HarEntriesList.tsx

```diff
@@ -1,17 +1,17 @@
-import {EntryItem} from "./EntryListItem/EntryListItem";
```

Comment:
sorry about those "little" renaming changes - it is probably happen because we started this branch before it was changed in frontend.
without HAR (only EntriesList) it is probably makes more sense